apps/common/src/main/scala/org/lfdecentralizedtrust/splice/automation/DomainParamsIngestionTrigger.scala

```diff
@@ -39,9 +36,12 @@ final class DomainParamsIngestionTrigger(
         Future.successful(TaskNoop)
       case Failure(e) => Future.failed(e)
       case Success(synchronizerId) =>
-        domainStateTopology
-          .firstAuthorizedStateForTheLatestSynchronizerParametersState(synchronizerId)
```

Comment:
I have no idea why this trigger fetched the history before. This just seems confusing and wasteful.